apps/minifront/src/fetchers/balances/by-account.ts

```diff
@@ -1,26 +1,25 @@
 import {
   Address,
-  AddressIndex,
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
 import { getBalances } from '.';
 import { BalancesResponse } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
 import { getAddress, getAddressIndex } from '@penumbra-zone/getters/src/address-view';
 
 export interface BalancesByAccount {
-  index: AddressIndex;
+  account: number;
```

Comment:
We should not use `AddressIndex` when grouping by account, because one account can have many ephemeral addresses, each of which will have a different `AddressIndex`

Before:

![image](https://github.com/penumbra-zone/web/assets/25391690/13629329-5277-4939-b013-f7a986896606)

After: 

<img width="642" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/345c1404-6112-45c7-8aa7-45f87903f1b4">